js/webpack/libraries.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     // library
     'inject': ['./inject.js'],
     'web3': ['./web3.js'],
-    'parity': ['./parity.js']
+    'parity': ['./library.parity.js']
```

Comment:
5da69d7 renames `parity.js` to `parity.npm.js` and reverts `library.parity.js` to `parity.js`.